<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Scene" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>341</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionEdition_setting"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New scene</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSingle_selection">
   <property name="text">
    <string>Single face selection</string>
   </property>
  </action>
  <action name="actionMultiple_face_selection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Multiple face selection</string>
   </property>
  </action>
  <action name="actionSingle_figure_selection">
   <property name="text">
    <string>Single figure selection</string>
   </property>
  </action>
  <action name="actionMultiple_figure_selection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Multiple figure selection</string>
   </property>
  </action>
  <action name="actionEdition_setting">
   <property name="text">
    <string>Edition setting</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Scene</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">scene.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
